apps/blockchain/lib/blockchain/account.ex

```diff
@@ -601,4 +601,21 @@ defmodule Blockchain.Account do
       %{acct | balance: 0}
     end)
   end
+
+  @doc """
+  Resets an account to the basic struct.
+
+  ## Examples
+
+      iex> state = MerklePatriciaTree.Trie.new(MerklePatriciaTree.Test.random_ets_db())
+      ...>   |> Blockchain.Account.put_account(<<0x01::160>>, %Blockchain.Account{balance: 100, code_hash: "abc"})
+      iex> state
+      ...> |> Blockchain.Account.reset_account(<<0x01::160>>)
+      ...> |> Blockchain.Account.get_account(<<0x01::160>>)
+      %Blockchain.Account{}
+  """
+  @spec clear_balance(EVM.state(), EVM.address()) :: EVM.state()
```

Comment:
wrong typespec

Reply:
Is it? I think it is right. Which part is wrong?

Reply:
the method is called `reset_account`. the typespec is for `clear_balance`

Reply:
hahahaa. I was looking at the specs themselves and didn't look at the name. Thanks!

Reply:
Updated in https://github.com/poanetwork/mana/pull/311/commits/08f862ab9eae240e29ff45fe1f41542c572fb984